### PR TITLE
Appcenter support added

### PR DESCRIPTION
**Pull request by mrndjo**

### CODE REVIEW

**Comment by mrndjo** [comment]
I am not sure why "yarn run link" command failed. https://travis-ci.org/danger/danger-js/jobs/532793498  @orta Can you please help me?

File: source/ci_source/providers/AppCenter.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  }
+
+  get pullRequestID(): string {
+    return this.default.prID
+  }
+
+  get repoSlug(): string {
+    if (
+      ensureEnvKeysExist(this.env, ["BUILD_REPOSITORY_NAME"]) &&
+      ensureEnvKeysExist(this.env, ["BUILD_REPOSITORY_NAME"])
```
Comment (f-meloni) [inline source/ci_source/providers/AppCenter.ts]: Should this be `BUILD_REPOSITORY_URI`?

**Comment by orta** [comment]
Can you add AppCenter to the readme too please, so people know it's supported?

**Comment by almedin-mrndic-sn** [comment]
> Can you add AppCenter to the readme too please, so people know it's supported?

Done.

**Comment by orta** [comment]
No idea what's up with the link - seems sporadic, so I'll just take it as red

**Comment by orta** [comment]
Thanks!

**Pull request merged**